sydent/http/servlets/replication.py

```diff
@@ -161,12 +161,17 @@ def _async_render_POST(self, request):
 
         # Process any new invite tokens
 
-        # New and updated invite tokens come is as lists instead of dictionaries
-        # They are wrapped in a dictionary. Extract that first
+        # Get the container dictionary of new and updated invites
         invite_tokens = inJson.get('invite_tokens', {})
```

Comment:
Can you add a comment showing the rough structure? `invite_tokens: { "added": { .. }, .. }` or something something